### PR TITLE
initial draft of DV didact

### PR DESCRIPTION
**Pull request by bfitzpat**
Signed-off-by: bfitzpat@redhat.com <bfitzpat@redhat.com>

### CODE REVIEW

**Comment by bfitzpat** [comment]
A few things noticed along the way...

* It would be nice to have "New" actions as inline options (like the "Edit" options) for Data source, DS properties, and Environment variables
* I am currently unable to leverage some of the deeper tree commands in Didact because they expect certain things to be in place (i.e. a tree selection I suspect) - we would need to modify some commands so they would work a bit better in Didact (but for now some text explaining what to do isn't bad)
* We should add some animated gifs showing what to do and that would help some of these non-automated steps

**Comment by bfitzpat** [comment]
Also, I found one niggly thing in the DV tree. The node is `Data Sources` but the new menu is `New Datasource` (all one word). Just a consistency thing.

File: didact/simple-dv-example.didact.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+First, we need to create our Virtual Database (VDB). 
+
+You can do this in a couple of ways:
+
+* Open the `Data Virtualization` view [(Execute^)](didact://?commandId=datavirt.focus){.didact}, hover over the view title, click `...` and choose `New Virtual Database`. [(Execute^)](didact://?commandId=datavirt.create.vdb){.didact}
+* Or access the same command from the Command Palette. Press `F1` or `Ctrl+Shift+P` to bring up the Command Palette, and type `New Virtual Database`. When the command is selected, click Enter. [(Execute^)](didact://?commandId=datavirt.create.vdb){.didact}
+
+Provide the name of your new VDB - `portfolio` - and press Enter.
+
+You should see a new file `portfolio.vdb` appear in your Explorer view and in the Data Virtualization view [(Execute^)](didact://?commandId=datavirt.focus){.didact}
```
Comment (lhein) [inline didact/simple-dv-example.didact.md:29]: it would be a portfolio.yaml file.

Comment (bfitzpat) [reply didact/simple-dv-example.didact.md:29]: ah yes. good catch

**Comment by bfitzpat** [comment]
@lhein do we want to close this since DV is on hold until further notice?

**Comment by lhein** [comment]
yes, close it please.

**Pull request closed without merging**